_posts/2016-03-09-public-speaking-part1-is-it-for-me.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Public Speaking - Part 1: Is It For Me?"
+date: 2016-03-09 11:00
+author: maxim
+categories: [teaching, learning, speaking, oss, community]
+---
+
+At Artsy, sharing knowledge is heavily encouraged and so we write our code in the open, document using blog posts, and speak about our experiences. Many have [bumped into us][tw_ldn] at various places around the world. And so I often get the question: “How did you get started? I’m interested in speaking too”.
+
+In the name of sharing experiences with others, I can’t encourage this enough. Unfortunately though, public speaking is still considered [man’s biggest fear][atc_fear].
+
+I’d like to share with you the benefits of public speaking, tips on how to get started, and how to give great presentations.
+
+<!-- more -->
+
+## Why do it?
+
+### Communication
+
+Without necessarily realising, you’re publicly speaking all the time. At the heart of it, it’s the act of communicating and expressing your ideas and viewpoints for others to understand.
```

Comment:
Not sure if you're interested in my tweet storm, but I ["wrote" about this](https://twitter.com/ashfurrow/status/705496590715203584) and how effective programming _is_ effective communication.